<?xml version="1.0" encoding="utf-8"?>

<launch>

  <!-- Load Parameter -->
  <param name="image_topic"       value="/camera/color/image_raw" />
  <param name="weight_path"       value="$(find yolov5_ros)/weights/orange.pt"/>
  <param name="pub_topic"         value="/yolov5/BoundingBoxes" />
  <param name="conf"              value="0.2" />

  <!-- Start yolov5 and ros wrapper -->
  <node pkg="yolov5_ros" type="yolo_v5.py" name="yolov5_ros_smoke" output="screen" />


</launch>
